### ISSUE TITLE
Prepare demo for FSI demo day Thursday Jan 12th

### ISSUE THREAD

**Issue by mkcomer** [microsoft/ccf-app-samples#98]
**FSI DEMO DAY is on Thursday, January 12th. This will not be our final demo. The purpose of FSI Demo day is to showcase what the team has been working on. We will have a final demo for the PO and other CCF/mCCF stakeholders at the end of January.** 

**MK requested a 20 min demo slot.** Please confirm with Narmatha Bala that Neutrino has 20 minutes for demo day on Thursday, January 12th @ross-p-smith or @macromania.

# Demo Outline

## Part 1 - Overview of Data Reconciliation and CCF

Assignees: TODO. I believe @macromania has interest in this. 

For fsi demo day, this overview can be captured and presented via markdown (powerpoint is not necessary). For this part, we will focus on the business application, why you would need to reconcile, how we reconcile, why you use CCF to reconcile (secure, multi-party compute), and the ledger. We can also introduce governance to FSI engineers :).

### Acceptance Criteria
- [ ] Create overview of data reconciliation and add markdown to demo folder. 

## Part 2 - Deploy, Ingest Data and Reconcile Data

Assignees: @kusweta 

For FSI Demo Day, we will plan to run our application in sandbox. Stretch goal - running on MCCF.
Minimally, we will demo 3 members ingesting data and first round of reconciliation (members query for reports). We may want to break the scripts up to make more sense to demo.

### Acceptance Criteria
- [ ] Create demo script and add to demo folder

## Part 3 - Demo a member adding additional data

Assignees: @kusweta 

Minimally, we will demo how a change in data changes the report. A member will submit more data to the network. When another member queries for their report, their report should look different than part 2.

### Acceptance Criteria
- [ ] Create demo script and add to demo folder

## (STRETCH) Part 4 Demo Governance - applying a code change 

Assignees: TODO. Blocked by https://github.com/orgs/microsoft/projects/542/views/1

This is a stretch goal for the fsi demo Here we will demo a regulatory change to the network. One member proposes a code change ( add `total number of votes` to reporting summary). ~All~ A majority of members vote on the change. Once accepted, the app is redeployed and when a member queries for a report again, we will see the additional data point (total number of votes).

### Acceptance Criteria
- [ ] Create demo script and add to demo folder

## Resources
- Data for fsi demo is [here](https://github.com/microsoft/ccf-app-samples/tree/main/data-reconciliation-app/demo/data)
- Leverage previous scripts to create a polished demo script: 
  Part 2 - [Reuse E2E testing script](https://github.com/microsoft/ccf-app-samples/issues/97)
  Part 4 - [Leverage governance script](https://github.com/microsoft/ccf-app-samples/issues/55)
